scripts/lego.sh

```
@@ -1,5 +1,11 @@
 #!/bin/sh
+set -a
+. ../etc/lego.env
```

Shellcheck (0.11.0)

[info] Not following: ../etc/lego.env was not specified as input (see shellcheck -x).

(SC1091)